### PR TITLE
Topbar

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
Summary

Top Bar implementation

Screenshot

captura de pantalla 2018-12-04 a la s 11 16 00 a m

Trello card

https://trello.com/c/Eck6yoWy

### CODE REVIEW

File: tic-tac-toe/src/app/components/Topbar/layout.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <Link to={url.src} key={index} className={style.link}>
+        {url.name}
+      </Link>
+    ));
+    return links;
+  };
+
+  render() {
+    const { urls, logoutClick } = this.props;
+    console.log(this.props);
```
Comment (julibel) [inline tic-tac-toe/src/app/components/Topbar/layout.js]: delete this console.log

File: tic-tac-toe/src/app/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import Login from '@screens/Login';
+import Profile from '@screens/Profile';
+import Topbar from '@components/Topbar';
+import { sessionValidation } from '@config/session';
+import { LOGGEDIN } from '@constants/const';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    sessionValidation(props.dispatch);
```
Comment (julibel) [inline tic-tac-toe/src/app/index.js]: as i marked in another PR.. we should not be using constructor. We will talk about it tomorrow

File: tic-tac-toe/src/app/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    );
+  }
+}
+
+const mapStateToProps = store => ({
+  loggedState: store.login.loggedState
+});
+
+App.propTypes = {
+  history: PropTypes.shape({}),
```
Comment (julibel) [inline tic-tac-toe/src/app/index.js]: idem other PRs, lets comeplete the shape

File: tic-tac-toe/src/app/screens/Game/index.js [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
+import { push } from 'connected-react-router';
+import { linesWin, LOGGEDIN } from '@constants/const';
 import { clickSquare, clickStep } from '@redux/Game/actions';
 
 import Game from './layout';
 
 class GameContainer extends Component {
+  constructor(props) {
+    super(props);
+    const { loggedState } = props;
```
Comment (julibel) [inline tic-tac-toe/src/app/screens/Game/index.js]: idem

**Review by julibel — CHANGES_REQUESTED** [review]
I marked some issues here and others.. are in other PRs 👼

**Comment by jeymarsonwolox** [comment]
Change session validation

File: tic-tac-toe/src/app/components/Topbar/layout.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  }
+}
+
+Topbar.propTypes = {
+  urls: PropTypes.arrayOf(
+    PropTypes.shape({
+      src: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired
+    })
+  ),
```
Comment (julibel) [inline tic-tac-toe/src/app/components/Topbar/layout.js:36]: is requiered?

**Review by julibel — CHANGES_REQUESTED** [review]
Perfect!! Just a comment..

File: tic-tac-toe/src/app/components/Topbar/layout.js (hunk 5 above)
Comment (julibel) [reply tic-tac-toe/src/app/components/Topbar/layout.js:36]: i seen it in other PR!! done.. this is good to go. Be careful with the merge 👀

**Review by julibel — APPROVED** [review]
Be careful when you merge! take into account that this is going to the other branch.. so i would correct all of them first, and when everything is approve, it can be merged in order. Just an idea.. feel free with this 😉

**Pull request merged**